corehq/apps/hqadmin/tests/test_stale_data_in_es.py

```diff
@@ -193,6 +196,20 @@ def call():
         self._send_cases_to_es([case])
         self._assert_in_sync(call())
 
+    def _test_case_missing_date(self):
+        def call():
+            return self._stale_data_in_es('case')
+        form, (case,) = self._submit_form(self.project.name, new_cases=1)
+
+        pg_modified_on = case.server_modified_on
+        case.server_modified_on = None
+        self._send_cases_to_es([case], refetch_doc=False)
```

Comment:
I'm assuming this line is where this test fails (prior to having the bug fixed). Is that right? What do you think of adding a comment to note that this is the operation being tested?

Reply:
Oh, I think I'm wrong. Maybe it's the `_stale_data_in_es` call that failed?

Reply:
Yeah it would be `_stale_data_in_es` that failed prior, it would have choked on the null date.